tests/build.gradle

```diff
@@ -16,6 +16,7 @@ tasks.withType(Test) {
 }
 
 task testLean(type: Test) {
+    exclude '**/*Java*'
```

Comment:
@markusthoemmes 
The images don't take much time to push (4 minutes)
But building the images now for python, swift takes more time and we go over the budget, even excluding *Java* I go out.

What's specific is your proposed alternative? 
I don't know what you mean by "Auto-building from Dockerhub" ?